dojo/forms.py

```diff
@@ -181,7 +181,7 @@ def __init__(self, *args, **kwargs):
 
     class Meta:
         model = Product
-        fields = ['name', 'description', 'tags', 'product_manager', 'technical_contact', 'team_manager', 'prod_type',
+        fields = ['name', 'description', 'tags', 'prod_manager', 'tech_contact', 'manager', 'prod_type',
```

Comment:
I missed this in my former review. The fields used in the product view page are product_manager, technical_contact and team_manager, so when adding a new product with these changes the "Team Manager" etc fields will show up as "Unknown".

When looking into this further it seems these fields introduced in this commit are marked as deprecated, personally I prefer keeping the old freetext fields rather than the 'new' ones that require these roles to be tied to users in the application.